### PR TITLE
Add ability to block/wait until standalone build succeeds or fails

### PR DESCRIPTION
**Pull request by mglagola**
@fson - Per our discussion to merge some of [exptool](https://github.com/mglagola/exptool) functionality with `exp`!

### CODE REVIEW

**Comment by fson** [comment]
This is awesome! Thanks for the pull request @mglagola.

Before starting to wait, we could show the "You can monitor the build at...<logs-url>" so people can go view the logs while waiting.

@anp had an idea that we could make this the default now, since it's almost always what you'd want. We can have `--no-wait`, which would revert back to the old behavior (exit once the build has started).

File: src/commands/build/BaseBuilder.js [hunk 1]
```diff
@@ -170,6 +174,38 @@ ${buildStatus.id}
...
+        fp.filter(job => job.id === buildId),
+        fp.getOr([], 'jobs')
+      )(res);
+      switch (job.status) {
+      case 'finished':
+        return job;
+      case 'pending':
+      case 'started':
+      case 'in-progress':
+        log(`Standalone app is still building, checking again in ${interval}s ...`);
```
Comment (fson) [inline src/commands/build/BaseBuilder.js]: This sometimes gets repeated quite a few times. Maybe we can just show a spinner and some message like this?
```
Waiting for build to complete. You can press Ctrl+C to exit.
```
We use the `@expo/simple-spinner` library for spinners, here's an example of how to use it: https://github.com/expo/exp/blob/95f81c99a163149da9c039148a50777a7a89614e/src/commands/diagnostics.js#L16-L20

**Comment by mglagola** [comment]
@fson Lmk what you think, added `simpleSpinner`, changed `--wait` to `--no-wait` and made sure `You can monitor the build at...` is always logged

**Review by fson — APPROVED** [review]
Looks great! I’m going to give this a final test in the morning and merge after that 👍

**Comment by fson** [comment]
Merged. Thanks again @mglagola, this makes the build command so much nicer!

**Pull request closed without merging**